draft-ietf-httpbis-priority.md

```diff
@@ -369,70 +388,77 @@ Stream ID:
 Priority Field Value:
 : The priority update value in ASCII text, encoded using Structured Headers.
 
-The HTTP/2 PRIORITY_UPDATE frame MUST NOT be sent prior to opening the
-stream.  If a PRIORITY_UPDATE is received prior to the stream being opened,
-it MAY be treated as a connection error of type PROTOCOL_ERROR.
+The PRIORITY_UPDATE frame MAY be sent before the stream that it references has
+been created. The Stream ID MUST be within the stream limit. If a server
+receives a PRIORITY_UPDATE for a Stream ID that is beyond the stream limits,
+this MUST be treated as a connection error of type PROTOCOL_ERROR.
```

Comment:
SHOULD.  I don't think that I could guarantee that this could be enforced.

```suggestion
this SHOULD be treated as a connection error of type PROTOCOL_ERROR.
```

Well, I guess that this is the h2 variant, but still.